### PR TITLE
Implements AAS and SM Repo integration with Registries feature

### PR DESCRIPTION
**Pull request by mdanish98**
Signed-off-by: Mohammad Ghazanfar Ali Danish <ghazanfar.danish@iese.fraunhofer.de>

### CODE REVIEW

File: basyx.submodelrepository/basyx.submodelrepository-feature-registry-integration/src/main/java/org/eclipse/digitaltwin/basyx/submodelrepository/feature/registry/integration/RegistryIntegrationSubmodelRepositoryFeature.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	}
+
+	@Override
+	public String getName() {
+		return "SubmodelRepository Registry Integration";
+	}
+
+	@Override
+	public boolean isEnabled() {
+		return true;
```
Comment (jannisjung) [inline basyx.submodelrepository/basyx.submodelrepository-feature-registry-integration/src/main/java/org/eclipse/digitaltwin/basyx/submodelrepository/feature/registry/integration/RegistryIntegrationSubmodelRepositoryFeature.java:72]: should only return true, when `basyx.submodelrepository.feature.registryintegration=$AAS_Registry_URL` and `basyx.externalurl=..` ist set.

**Review by jannisjung — COMMENTED** [review]
Please resolve the review remarks

File: basyx.aasrepository/basyx.aasrepository-feature-registry-integration/src/main/java/org/eclipse/digitaltwin/basyx/aasrepository/feature/registry/integration/RegistryIntegrationAasRepositoryFeature.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	}
+
+	@Override
+	public String getName() {
+		return "AasRepository Registry Integration";
+	}
+
+	@Override
+	public boolean isEnabled() {
+		return true;
```
[the remark this answers is not in the document]
Comment (mdanish98) [reply basyx.aasrepository/basyx.aasrepository-feature-registry-integration/src/main/java/org/eclipse/digitaltwin/basyx/aasrepository/feature/registry/integration/RegistryIntegrationAasRepositoryFeature.java]: Updated this in the latest commit.

File: basyx.submodelrepository/basyx.submodelrepository-feature-registry-integration/src/main/java/org/eclipse/digitaltwin/basyx/submodelrepository/feature/registry/integration/RegistryIntegrationSubmodelRepositoryFeature.java (hunk 1 above)
Comment (mdanish98) [reply basyx.submodelrepository/basyx.submodelrepository-feature-registry-integration/src/main/java/org/eclipse/digitaltwin/basyx/submodelrepository/feature/registry/integration/RegistryIntegrationSubmodelRepositoryFeature.java]: Updated this in the latest commit.

**Review by jannisjung — APPROVED** [review]

**Pull request merged**